principles/coding-principles.md

```diff
@@ -37,10 +38,9 @@ Comments are for explaining <strong>why</strong> something is needed, not how it
 Commit messages should follow [GDS guidance](https://github.com/alphagov/styleguides/blob/master/git.md)
 No-one cares how clever you are - it’s far more important to work well with your team
 APIs are interfaces too - Like any other interface, APIs need designing and iterating for usability
-Don’t pollute the global namespace
 
 ### 6. Think smaller
-Stick to the Single Responsibility Principle - keep view templates, controllers and model classes as simple as possible. Keep methods short. (Concepts and patterns which may help with this: Null Object pattern, Facade pattern, Form Objects, Sandi Metz’s “[Rules for Developers](https://robots.thoughtbot.com/sandi-metz-rules-for-developers)”)
+Stick to the Single Responsibility Principle - keep views, controllers and model classes as simple as possible. Keep methods short. (Concepts and patterns which may help with this: Null Object pattern, Facade pattern, Form Objects, Sandi Metz’s “[Rules for Developers](https://robots.thoughtbot.com/sandi-metz-rules-for-developers)”)
```

Comment:
I don't see much evidence of the 'Rules for developers' being followed...do you? I'm assuming this is a goal and not the 'current view'.

Reply:
This is "our expectations" of how development is to be carried out.

Reply:
For me the Rules for Developers of '5 lines per method' is a little extreme. Its a good goal but not so good as a rule (in my view). I'd just be worried that it might turn people off to all the rules???